pdr_backend/lake/lake_info.py

```diff
@@ -47,25 +54,76 @@ def print_table_info(self, source: Dict[str, DataFrame]):
             print(",".join(columns))
 
             if has_timestamp:
+                min_timestamp = source[table_name]["timestamp"].min()
                 max_timestamp = source[table_name]["timestamp"].max()
-                if max_timestamp is not None:
-                    print("Max timestamp: " + str(max_timestamp))
+
+                if isinstance(min_timestamp, (int, float)):
+                    min_datestr = datetime.fromtimestamp(
```

Comment:
please use `min_datestr = UnixTimeMs(min_timestamp).to_timestr()` and similarly for max. It is much more safe.